### PR TITLE
Minor fixes to make a few HyQL queries work

### PR DESCRIPTION
**Pull request by renan-souza**
The following [part of the code](https://github.com/ibm-hyperknowledge/hkpy/pull/20/commits/1fdcd3da3520e876f95295bf6d15e40581b2bb49#diff-96ee5d7b4f947ee6c45cb59a38972c418a67ffb8d48a7a305233c95a620a696fL245) always expected that some of the `e` were dict, but they were simple literals. The hkfy function was breaking when that happened.
```
row = [hkfy(e) for e in entry]
```

### CODE REVIEW

**Review by eltonfss — CHANGES_REQUESTED** [review]
@renan-souza can you provide a description of the objetive of this change?

**Comment by eltonfss** [comment]
> @renan-souza can you provide a description of the objetive of this change?

Please also change the title accordingly

File: hkpy/hkbase/hkrepository.py [hunk 1]
```diff
@@ -242,7 +242,15 @@ def hyql(self, query: str, transitivity: Optional[bool] = False) -> HKEntityResu
...
-            row = [hkfy(e) for e in entry]
+            if isinstance(entry, list):
+                row = []
+                for e in entry:
+                    if isinstance(e, dict):
+                        row.append(hkfy(e))
+                    else:
+                        row.append(e)
+            else:
+                row = entry
```
Comment (raphael-melo) [inline hkpy/hkbase/hkrepository.py:253]: Is the `cast` in line 240 still valid? It makes seem like there is only two options, list of list of dicts or dicts. I'm gathering that maybe there are HyQL that return a single column with literals. It that it?

Comment (raphael-melo) [reply hkpy/hkbase/hkrepository.py:253]: It that's the case, I'd suggest adding these to the the Union in line 240.

Comment (renan-souza) [reply hkpy/hkbase/hkrepository.py:253]: Fixed in: https://github.com/ibm-hyperknowledge/hkpy/commit/7bc08234892f20863c8934256af6a1d8e67a6734

**Review by raphael-melo — APPROVED** [review]

**Review by eltonfss — APPROVED** [review]

**Pull request merged**